src/omv/ports/stm32/omv_portconfig.mk

```
@@ -623,6 +623,7 @@ UVC_OBJ += $(addprefix $(BUILD)/$(OMV_DIR)/imlib/,\
 	)
 
 UVC_OBJ += $(addprefix $(BUILD)/$(OMV_DIR)/ports/stm32/,\
+	omv_gpu.o                               \
```

```suggestion
	omv_gpu.o                               \
```
At the end not here.

---

Done